src/js/playlist/source.ts

```diff
@@ -10,13 +11,34 @@ import { trim, extension } from 'utils/strings';
  * @property {string} label - The quality label to be used with multiple mp4/webm sources.
  */
 
-const Source = function(config) {
+export type PlaylistItemSource = PlaylistItemConfig;
```

Comment:
i opted to export this since naming wise it makes more sense, but i think having the internal `config` naming makes sense in terms of the actual application usage

Reply:
That's good, but `type` and `default` properties are not optional on `source`.

Reply:
ah nice catch, fixed